keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -1491,7 +1504,17 @@ kpxc.fillIn = function(combination, onlyPassword, suppressWarnings) {
         }
         if (pField) {
             pField.setAttribute('type', 'password');
-            kpxc.setValueWithChange(pField, kpxc.credentials[0].password);
+            let password = kpxc.credentials[0].password;
+            let totp = '';
```

Comment:
No need to define the variable here. You can just use `const totp = kpxc.getTOTP(kpxc.credentials[0]);` inside the if sentence.